.github/workflows/deploy-prod.yml

```
@@ -80,27 +80,27 @@ jobs:
 
       - name: Deploy documentation CloudFormation stack
         run: pnpm --filter="@tools/cdk" docs deploy
-#  deploy-services:
-#    name: Deploy services CloudFormation stack and production environment
-#
-#    runs-on: ubuntu-latest
-#
-#    environment:
-#      name: production-services
-#
-#    steps:
-#      - name: Checkout repository
-#        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4
-#
-#      - name: Setup Node.js and pnpm
-#        uses: ./.github/actions/setup-node-and-pnpm
-#
-#      - name: Install dependencies
-#        # Ensure NODE_ENV != production so pnpm will install devDependencies!!!
-#        run: NODE_ENV=development pnpm install --frozen-lockfile
-#
-#      - name: Build services
-#        run: pnpm build --filter="@services/*"
-#
-#      - name: Deploy stack
-#        run: pnpm --filter="@tools/cdk" services deploy
+  deploy-services:
+    name: Deploy services CloudFormation stack and production environment
+
+    runs-on: ubuntu-latest
+
+    environment:
+      name: production-services
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4
+
+      - name: Setup Node.js and pnpm
+        uses: ./.github/actions/setup-node-and-pnpm
+
+      - name: Install dependencies
+        # Ensure NODE_ENV != production so pnpm will install devDependencies!!!
+        run: NODE_ENV=development pnpm install --frozen-lockfile
```

I was thinking of opening a PR with this new strat 💯 

```suggestion
        run: pnpm install --frozen-lockfile --prod false
```

https://pnpm.io/cli/install#--prod--p

---

Per #89, eventually we want to make all root-level devDeps full dependencies so we don't have to add shims at all.